src/Modules/ContentBlocks/Domain/ContentBlock/ContentBlock.php

```diff
@@ -0,0 +1,172 @@
+<?php
+
+namespace ForkCMS\Modules\ContentBlocks\Domain\ContentBlock;
+
+use Doctrine\ORM\Mapping as ORM;
+use ForkCMS\Core\Domain\Settings\EntityWithSettingsTrait;
+use ForkCMS\Modules\Backend\Domain\Action\ModuleAction;
+use ForkCMS\Modules\Backend\Domain\User\Blameable;
+use ForkCMS\Modules\Internationalisation\Domain\Locale\EntityWithLocaleTrait;
+use DateTimeImmutable;
+use Pageon\DoctrineDataGridBundle\Attribute\DataGrid;
+use Pageon\DoctrineDataGridBundle\Attribute\DataGridActionColumn;
+use Pageon\DoctrineDataGridBundle\Attribute\DataGridMethodColumn;
+use Pageon\DoctrineDataGridBundle\Attribute\DataGridPropertyColumn;
+
+#[ORM\Entity(repositoryClass: ContentBlockRepository::class)]
+#[ORM\Table(name: 'contentblocks__contentblock')]
```

Comment:
Why not `ContentBlocks` like in default Doctrine?

Reply:
https://dev.to/ovid/database-naming-standards-2061
But implementing it as a naming strategy is probably a better idea: https://www.doctrine-project.org/projects/doctrine-orm/en/2.16/reference/namingstrategy.html
I've personally had multiple times where the camel cased table names gave issues when reinstalling fork leaving shadow tables behind leading me to have to remove the complete data directory for that database manually

Reply:
k, if it is by "convention" is it ok for me. 
But it should be documented or be consistent through Fork.
Or just follow symfony standards?

Reply:
https://github.com/doctrine/orm/blob/2.16.x/lib/Doctrine/ORM/Mapping/UnderscoreNamingStrategy.php
It is available in the codebase of doctrine as well.
In the other entities so far it's been configured manually as well so setting this should make it possible to remove that.
We can document it but setting it like this should automatically apply it everywhere

Reply:
@davysumo If I'm not wrong Jelmer already fixed this in the upstream. So I think this can be removed.

Reply:
Should be automatic now indeed